build_component.py

```diff
@@ -321,16 +313,29 @@ def main():
       # Build instructions for unit tests look like this:
       # "test ../relative/path/to/required/file_or_fileglob"
       if repytest:
-        source_spec = line.split(" ", 1)[1].strip()
+        source_spec = line.split()[1].strip()
+        try:
+          sub_target_dir = line.split()[2].strip()
```

Comment:
We should mention this subdirectory feature in the header block. A comment explaining why subdirectories are needed would help.